symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -349,6 +350,21 @@ public function appendAttachment($file)
     }
 
     /**
+     * Sets the property `$_ignore_attachment_errors`
+     *
+     * This property is false by default, so sending will break if any attachment
+     * can not be loaded; if it is true, attachment errors error will be ignored.
+     *
+     * @since Symphony 2.7
+     * @param boolean $ignore_attachment_errors
+     * @return void
+     */
+    public function setIgnoreAttachmentErrors(bool $ignore_attachment_errors)
```

Comment:
The type declaration `bool` only works on PHP7, so I will have to remove this. I will send a different PR. :-(